src/main/java/seedu/address/model/person/Id.java

```diff
@@ -22,19 +22,20 @@
     /**
      * Constructs an {@code Address}.
      *
-     * @param address A valid address.
+     * @param id A valid address.
      */
-    public Address(String address) {
-        requireNonNull(address);
-        checkArgument(isValidAddress(address), MESSAGE_CONSTRAINTS);
-        value = address;
+    public Id(String id) {
+        requireNonNull(id);
+        checkArgument(isValidId(id), MESSAGE_CONSTRAINTS);
+        value = id;
     }
 
     /**
      * Returns true if a given string is a valid email.
      */
-    public static boolean isValidAddress(String test) {
-        return test.matches(VALIDATION_REGEX);
+    public static boolean isValidId(String test) {
+        String[] idArray = test.split("");
+        return ((idArray.length == 9) && idArray[0].equals("A") && test.matches(VALIDATION_REGEX));
```

Comment:
As of now, this is good to check the id 💯 but remember to validate for different types of ids in the upcoming milestones to match our project definition of patron (can consist of students and staff).